src/main/java/nl/hsac/fitnesse/util/iban/BEIban.java

```diff
@@ -0,0 +1,232 @@
+package nl.hsac.fitnesse.util.iban;
+
+import nl.hsac.fitnesse.util.RandomUtil;
+
+/**
+ * Generates a Belgium IBAN.
+ */
+
+public class BEIban {
+    private RandomUtil randomUtil = new RandomUtil();
+    private IbanUtil ibanUtil = new IbanUtil();
+
+    /**
+     * Generates random number to create IBAN.
+     *
+     * @return random Belgian IBAN.
+     */
+
+    public String generateBEIban(String country, String bankCode) {
+
+        if (country.equals("")) {
```

Comment:
Why does one need to supply a country code to this class? Is there any chance another value than `BE` would be desired? (this goes for all country specific classes)